src/components/BrandLogo/_BrandLogo.scss

```diff
@@ -27,27 +27,14 @@ $border-radius: 2px;
     }
 
     .right {
```

Comment:
Why did you change all of this?

The logo does not look nice anymore, since right now there is hardly any padding around the IO part -- it looks messy.

Reply:
What do you think these code do? Are you sure they have anything to do with paddding?
Have you ever compared the difference before and after my commit?